Tests/CartonCommandTests/DevCommandTests.swift

```diff
@@ -16,8 +16,11 @@
 //
 
 @testable import CartonCLI
+import Path
```

Comment:
We don't need this `Path` module after all, do we? If so, could it be removed from `Package.swift` as well?